source/Test.cpp

```diff
@@ -457,7 +457,7 @@ void Test::Step(TestContext &context, PlayerInfo &player, Command &commandToGive
 			case TestStep::Type::INJECT:
 				{
 					// Lookup the data and inject it in the game or into the environment.
-					const TestData* testData = (GameData::TestDataSets()).Get(stepToRun.nameOrLabel);
+					const TestData *testData = (GameData::TestDataSets()).Get(stepToRun.nameOrLabel);
```

Comment:
Are these brackets necessary?
```suggestion
					const TestData *testData = GameData::TestDataSets().Get(stepToRun.nameOrLabel);
```